chutney/ui/src/app/modules/scenarios/components/execution/history/list/scenario-executions.component.html

```diff
@@ -169,11 +182,17 @@
               <td>
                 <ngb-highlight [result]="execution.user" [term]="filtersForm.get('keyword').value"></ngb-highlight>
               </td>
+                <td>
+                    <button class="btn btn-sm btn-primary me-2" (click)="replay(execution, $event)"
+                            [ngbTooltip]="'global.actions.replay' | translate" placement="left">
+                        <i  class="bi bi-repeat" ></i>
```

Comment:
Suprised by the use of bootstrap icon. We normally use fontawesome.
We got to maybe choose which one we want to use.

Reply:
there is an item about that in the backlog